x/csr/types/msg.go

```diff
@@ -49,15 +57,15 @@ func (msg MsgRegisterCSR) ValidateBasic() error {
 	eg := &errgroup.Group{}
 	// check that the deployer is a valid canto address
 	if _, err := sdk.AccAddressFromBech32(msg.Deployer); err != nil {
-		return sdkerrors.Wrapf(err, "MsgRegisterCSR: ValidateBasic: invalid sdk address: %s", msg.Deployer)
+		return sdkerrors.Wrapf(err, "MsgRegisterCSR: ValidateBasic: invalid sdk address of deployer: %s", msg.Deployer)
```

Comment:
```suggestion
		return sdkerrors.Wrapf(err, "MsgRegisterCSR::ValidateBasic invalid sdk address of deployer: %s", msg.Deployer)
```
per convo on slack we need to conform to same error structure